components/fields/DependentField.js

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from "react";
+import { useFormikContext, useField } from "formik";
+
+const DependentField = (props) => {
+  const { values, setFieldValue } = useFormikContext();
+  const [field] = useField(props);
+
+  const { format, inputToMirror, name } = props;
+  const value = values[inputToMirror];
+
+  // Remove inputToMirror & format props because it is not a legit input
+  // attribute and it will give us a warning of we put it on an input element
+  const propKeys = Object.keys(props);
+  const initialValue = {};
+  const filteredPropsReducer = (acc, curr) => {
+    if (curr === "inputToMirror" || curr === "format") return acc;
+
+    const newObj = { ...acc, [curr]: props[curr] };
+
+    return newObj;
+  };
```

Comment:
Would this produce the same result?
```suggestion
  const { format, inputToMirror, ...filteredProps } = props;  
```

Reply:
This produces the same result :). I forgot how spreading works. Will work on it.